### ISSUE TITLE
Shift+Tab to access block toolbar doesn't work in Safari

### ISSUE THREAD

**Issue by ZebulanStanphill** [WordPress/gutenberg#22598]
## The issue
On Safari, pressing Shift-Tab to move focus from the block content to the block toolbar doesn't work. Instead, the address bar of the browser is focused.

## How to reproduce
1. Open the block editor in Safari.
2. Create a Paragraph block (or some other basic text block).
3. Focus the content of the block.
4. Press <kbd><kbd>Shift</kbd>+<kbd>Tab</kbd></kbd>.
5. Notice that the address bar is focused, not the block toolbar like you would expect.

## Additional info

Tested on `master` as of https://github.com/WordPress/gutenberg/commit/7f797a6bd9e88d8348cd9807da71ad2cc85e3886. I don't know when this regressed, but I suspect it may have started with #20008.

**Comment by enriquesanchez** [comment]
Hi @ZebulanStanphill!

Does this only happen on Safari? Could it maybe be related to https://github.com/WordPress/gutenberg/issues/22434?

**Comment by ZebulanStanphill** [comment]
I tested: it doesn't happen on Firefox, nor does it happen on Chrome. (I tested both on macOS.) The inserter was not open in my tests, so I don't think it's related.

**Comment by enriquesanchez** [comment]
Hey @ZebulanStanphill 👋

I can't reproduce on Safari, running tip of `master`:

![2020-05-28 14-59-39 2020-05-28 15_00_54](https://user-images.githubusercontent.com/3276087/83187655-1b609400-a0f4-11ea-84fe-757b012ed771.gif)

**Comment by ZebulanStanphill** [comment]
This is still happening to me. Tested in Safari 13.1 on macOS Catalina 10.15.4 using Gutenberg `master` as of https://github.com/WordPress/gutenberg/commit/ed8029923fc9bc83f2cdfedca60225188aae9f17.

**Comment by ZebulanStanphill** [comment]
I just discovered that this isn't a bug. It's just [standard default behavior](https://support.apple.com/en-ca/guide/safari/cpsh003/mac) in Safari.

>Tab highlights the next text field or pop-up menu. Tab also highlights buttons and other controls if “All controls” is selected in the Shortcuts pane of the Keyboard pane of System Preferences.
>
>Option-Tab highlights the same items as Tab plus all other clickable items.
>
>To swap the behavior of Tab and Option-Tab, turn on “Press Tab to highlight each item on a webpage” in the Advanced pane of Safari preferences.

In other words you have to use <kbd><kbd>Shift</kbd>+<kbd>Option</kbd>+<kbd>Tab</kbd></kbd> to achieve the same thing in Safari as <kbd><kbd>Shift</kbd>+<kbd>Tab</kbd></kbd> in other browsers, unless you change Safari settings.
